ui-components/src/locales/general.json

```diff
@@ -967,7 +967,7 @@
       "accountHasBeenLocked": "For security reasons, this account has been locked.",
       "youHaveToWait": "You’ll have to wait 30 minutes since the last failed attempt before trying again.",
       "enterValidEmailAndPassword": "Please enter a valid email and password.",
-      "enterValidEmailAndPasswordAndMFA": "Please enter a valid email, password, and code.",
+      "enterValidEmailAndPasswordAndMFA": "Please enter a valid code",
```

Comment:
Sorry to ask for such a small change but all of the other similar error strings end in a period - after that good to merge 🥳